docs/ReleaseNotes.md

```diff
@@ -19,7 +19,9 @@ The included licenses apply to the following files:
 
 ### Upcoming Release
```

Comment:
Any reason not to just update the "Upcoming Release" heading as part of this PR?

Reply:
I was waffling on that. Updating it makes it not a true cherry-pick. If we are okay with that then we should update it here.

The other consideration is how cleanly it'll merge back into main. I _think_ having this as a true cherry-pick means it can be ignored when merging back into main but I can't say for sure